README.md

```diff
@@ -36,25 +36,16 @@ Eventually you should have the following folder structure:
 If you want to use another folder, specify the `dataroot` parameter of the NuScenes class below.
 
 ## Devkit setup
-Download the devkit to your home directory using:
-```
-cd && git clone https://github.com/nutonomy/nuscenes-devkit.git
-```
-The devkit is tested for Python 3.5.
-To install Python 3.5 and set up a new virtual environment, you can look at [these instructions](#setting-up-a-new-virtual-environment).
-To install the required packages, run the following command in your favourite virtual environment:
-```
-pip install -r requirements.txt
-```
-Also add the `python-sdk` directory to your `PYTHONPATH` environmental variable, e.g. by adding the 
-following to your `~/.virtualenvs/nuscenes/bin/postactivate` (virtual environment) or `~/.bashrc` (global):
-```
-export PYTHONPATH="${PYTHONPATH}:$HOME/nuscenes-devkit/python-sdk"
-```
-Finally, set NUSCENES env. variable that points to your data folder
+The devkit is tested for Python 3.5. To install python, please check [here](https://github.com/nutonomy/nuscenes-devkit/blob/master/installation.md#install-python).
+It is then recommended to install the devkit in a new virtual environment. To set-up a new virtual environment, please check [here](https://github.com/nutonomy/nuscenes-devkit/blob/master/installation.md#setup-a-new-virtual-environment).
```

Comment:
Make it clear that these are two alternatives (in that order):
- Either you directly download the pip package.
- Or you setup a virtual environment.